README.md

```diff
@@ -1,184 +1,86 @@
-| :--------------------------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------: |
-| [<img src="https://avatars.githubusercontent.com/u/80308339?v=4" width = "200" />](https://github.com/mariuscmoldovan) | [<img src="https://avatars.githubusercontent.com/u/64490927?v=4" width = "200" />](https://github.com/Spencerp34) | [<img src="https://avatars.githubusercontent.com/u/82188608?v=4" width = "200" />](https://github.com/ray-rafe) | [<img src="https://avatars.githubusercontent.com/u/84108714?v=4" width = "200" />](https://github.com/johanna-rodriguez) | [<img src="https://avatars.githubusercontent.com/u/16638790?v=4" width = "200" />](https://github.com/rickmansfield) |
-|              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/mariuscmoldovan)              |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/Spencerp34)              |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/ray-rafe)              |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/johanna-rodriguez)              |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/rickmansfield)              |
-|                                                       Front End                                                        |                                                     Back End                                                      |                                                      UX/UI                                                      |                                                          UX/UI                                                           |                                                         TPM                                                          |
+```bash
+  npm install
+```
 
-<br>
-<br>
+Start the server
```

Comment:
It might be helpful to have a hint that many of the features of the FE won't work correctly unless you have the BE server running as well